pyrdp/mitm/DeviceRedirectionMITM.py

```diff
@@ -325,6 +327,10 @@ def sendForgedFileRead(self, deviceID: int, path: str) -> int:
         :param path: path of the file to download. The path should use '\' instead of '/' to separate directories.
         """
 
+        if not self.config.extractFiles:
+            self.log.warning('Ignored attempt to forge file reads because file extraction is disabled.')
```

Comment:
I think warning is a too intense log level for this. Since the feature was activated by the user, info would suffice.

Reply:
```suggestion
            self.log.info('Ignored attempt to forge file reads because file extraction is disabled.')
```